docs/book/src/reference/completion.md

```diff
@@ -0,0 +1,53 @@
+# Enabling shell auto completion
```

Comment:
I am not able to check it in the Reference category in the preview. Could you please check?

<img width="293" alt="Screenshot 2020-03-26 at 11 21 36" src="https://user-images.githubusercontent.com/7708031/77641485-0cbefa80-6f54-11ea-9a63-349d84e156a7.png">